exoctk/exoctk_app/app_exoctk.py

```diff
@@ -482,19 +482,22 @@ def contam_visibility():
 
             # Make plot
             title = form.targname.data or ', '.join([form.ra.data, form.dec.data])
-            pG, pB, dates, vis_plot, table = vpa.using_gtvt(str(form.ra.data), str(form.dec.data), form.inst.data.split(' ')[0])
+            pG, pB, dates, vis_plot, table, badPAs = vpa.using_gtvt(str(form.ra.data),
+                                                                    str(form.dec.data),
+                                                                    form.inst.data.split(' ')[0],
+                                                                    targetName=str(title))
 
             # Make output table
-            vers = '0.3'
+            vers = '1.0'
             today = datetime.datetime.now()
             fh = StringIO()
-            fh.write('# Hi! This is your Visibility output file for... \n')
-            fh.write('# Target: {} \n'.format(form.targname.data))
-            fh.write('# Instrument: {} \n'.format(form.inst.data))
-            fh.write('# \n')
-            fh.write('# This file was generated using ExoCTK v{} on {} \n'.format(vers, today))
-            fh.write('# Visit our GitHub: https://github.com/ExoCTK/exoctk \n')
-            fh.write('# \n')
+            #fh.write('# Hi! This is your Visibility output file for... \n')
```

Comment:
Why are these all commented out?

Reply:
@catherine-martlin this text made the `.csv` file a lot clunkier, and wasn't necessary. I'll remove them